### ISSUE TITLE
A complete active request should be removed from the active request list once the booking time has passed

### ISSUE THREAD

**Issue by daniyalb** [utmgdsc/hacklab-booking#45]

**Comment by logonoff** [comment]
implementation: should this only affect what the user sees in the frontend?

**Comment by daniyalb** [comment]
yeah, we could add another status to requests like "done" once the endDate passes so it no longer shows up in a user's active requests

**Comment by logonoff** [comment]
rather than store redundant data we could also filter out past events when the client makes a GET request to the backend. this is less computationally efficient though

**Comment by daniyalb** [comment]
I think that’s a good idea, even if we had extra status data we would still
need to filter the requests, so we might as well keep it simple by just
checking the endTime in the backend route to get requests

**Comment by ggggg** [comment]
Don't know if this is still an issue with the new backend since the new backend only sends requests in the future by default, unless asked to do otherwise.
